release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -112,7 +112,7 @@ This change does not affect machine config pool rollout duration.
 [id="ocp-4-8-mco-upgrade-complete"]
 ==== MCO waits for all machine config pools to update before reporting the update is complete
 
-When updating, the Machine Config Operator (MCO) now reports an `Upgradeable=False` status to the Cluster Version Operator (CVO) if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported the `Upgradeable` status based only on the state of the master machine config pool, even if the worker pools were not done updating.
+When updating, the Machine Config Operator (MCO) now reports an `Upgradeable=False` condition in its Cluster Operator if any machine config pool has not completed updating. This status blocks future minor updates, but does not block future patch updates, or the current update. Previously, the MCO reported the `Upgradeable` status based only on the state of the master machine config pool, even if the worker pools were not done updating.
```

Comment:
"condition in the", instead of "its"-avoid possession

Reply:
@skrthomas I left it as _it`s_ as each operator (I've learned) has it's own Cluster Operator. 
How about _in the machine-config Cluster Operator_ ? 
An Atom search yields a few examples, etc Cluster Operator and  console Cluster Operator. Both _etcd_ and _console_ appear to be the name of the operator (oc get clusteroperator) and do not take mark-up.

Reply:
I like that!